chromium_src/chrome/browser/media/router/media_router_feature.cc

```diff
@@ -14,12 +14,12 @@
 namespace media_router {
 
 bool MediaRouterEnabled(content::BrowserContext* context) {
-#if !defined(OS_ANDROID)
-  if (GetMediaRouterPref(context)->IsDefaultValue()) {
-    return false;
-  }
-#endif
+#if defined(OS_ANDROID)
   return MediaRouterEnabled_ChromiumImpl(context);
+#endif
+  const PrefService::Preference* pref = GetMediaRouterPref(context);
```

Comment:
this is going to bypass the feature enable/disable check

Reply:
discussed via DM, feature flag for media router is not exposed in brave://flags

Reply:
That doesn't mean you can ignore it here, you still have to check `base::FeatureList::IsEnabled(kMediaRouter)`

Reply:
Updated!